packages/twenty-front/src/modules/ui/utilities/state/component-state/hooks/useScopeIdFromStateContext.ts

```diff
@@ -0,0 +1,23 @@
+import { useScopeInternalContext } from '@/ui/utilities/recoil-scope/scopes-internal/hooks/useScopeInternalContext';
+import { ComponentFamilyState } from '@/ui/utilities/state/component-state/types/ComponentFamilyState';
+import { ComponentState } from '@/ui/utilities/state/component-state/types/ComponentState';
+
+export const useScopeIdFromStateContext = (
```

Comment:
useComponentId!

Reply:
It's not componentId it's scopeId, that's not the same, scopeId is prefixed with "-scope-id".